plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -770,6 +774,7 @@ class RpmBase(NonMetadataPackage):
     summary = mongoengine.StringField()
     time = mongoengine.IntField()
     requires = mongoengine.ListField()
+    recommends = mongoengine.ListField()
```

Comment:
please unittest here

Reply:
I checked the unittests and unless I'm missing something, [these fields are not being unittested](https://github.com/pulp/pulp_rpm/blob/2-master/plugins/test/unit/plugins/db/test_models.py#L975,#L995) IMO just the basic mongoengine functionality would be exercised by it so I'm unsure about value added.